src/template.rs

```diff
@@ -0,0 +1,56 @@
+use handlebars::{handlebars_helper, Handlebars};
+use rand::seq::SliceRandom;
+
+pub fn addhelpers(x: &mut Handlebars) {
+    handlebars_helper!(wrap: |index: usize, s: String| {
+        let mut data = s.clone();
+        if s.len() > index + 1 {
+            data.insert(index, '\n')
+        } else {
+            data.push('\n');
```

Comment:
I would think it to just return string as it is, if length of string is less than index, but i want to hear your thoughts about it.